src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -20,60 +20,59 @@
 #
 # FIXME: Replace the following function calls with calls to functions that emit full standalone samples. These stubs have been adapted from method_sample.snip
 @snippet standaloneSample(apiMethod, sample)
-  {@standaloneSampleSetup(apiMethod, sample)}
-  @switch sample.callingForm
-  @case "Request"
-    {@optionalArrayMethodSampleCodeNonStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "RequestPaged"
-    {@pagedOptionalArrayMethodSampleCodePaged(apiMethod, sample)}
-  @case "RequestPagedAll"
-    {@pagedOptionalArrayMethodSampleCodeAll(apiMethod, sample)}
-  @case "RequestStreamingBidi"
-    {@optionalArrayMethodSampleCodeBidiStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "RequestStreamingClient"
-    {@optionalArrayMethodSampleCodeClientStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "RequestStreamingServer"
-    {@optionalArrayMethodSampleCodeServerStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "LongRunningPromise"
-    {@lroSampleCode(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @default
-    $unhandledCallingForm: {@sample.callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
-  @end
-  
-  {@standaloneSampleTeardown(apiMethod, sample)}
-@end
-
-@private standaloneSampleSetup(apiMethod, sample)
   @# [START {@sample.regionTag}]
-  
+
   {@importList(sample.initCode.importSection.appImports)}
 
-  @# [START {@sample.regionTag}_core]
-  
-  client = {@apiMethod.apiModuleName}.{@apiMethod.apiClassName}()
+  def some_func({@sampleFuncParam(sample.initCode.argDefaultParams)}):
```

Comment:
We should call the sample's function something like `sample_{@apiMethod.name}`

Reply:
Done